pyproject.toml

```
@@ -285,7 +285,7 @@ dl = [
   'tensorflow<2.17,>=2; python_version < "3.12"',
   'torch; python_version < "3.12"',
   'transformers[torch]<4.41.0; python_version < "3.12"',
-  'pykan; python_version > "3.9.7"',
+  'pykan<0.3>=0.2; python_version > "3.9.7"',
```

```suggestion
  'pykan<0.3,>=0.2; python_version > "3.9.7"',
```

Sorry, I missed a comma.